### PR TITLE
Django22 Support

### PR DESCRIPTION
**Pull request by Ayub-Khan**
-Updated Tox
-Updated travis
-Updated trove classifiers
-removed deprecated pep8 library

### CODE REVIEW

**Comment by Ayub-Khan** [comment]
fyi: https://tools-edx-jenkins.edx.org/job/translations/job/edx-platform-pull_translations/181/
Those are some known issues. I created a ticket long time ago to fix those, might be some in spartans backlog. Until those issues are fixed we can not use django20+ for translation jobs.

File: .travis.yml [hunk 1]
```diff
@@ -8,17 +8,24 @@ python:
   - 3.6
 
 env:
-  - TOXENV=django18
-  - TOXENV=django19
-  - TOXENV=django110
   - TOXENV=django111
+  - TOXENV=django20
+  - TOXENV=django21
+  -- TOXENV=django22
```
Comment (awais786) [inline .travis.yml]: typo `--`

**Comment by jmbowman** [comment]
@Ayub-Khan Could you find those tickets and link to them here?  It looks like we're on a pretty old version of the Babel package, would upgrading that help?  It would help if the tests captured `stderr` from the command that failed instead of just saying that the exit status was non-zero.

**Comment by Ayub-Khan** [comment]
@jmbowman looking into this. Im not sure, i was testing with the help of some one else i do not have access to translation jobs.

**Comment by Ayub-Khan** [comment]
@jmbowman after having another look i think we should move forward with this release first and come back to pybabel issue when we are working on edx-platform translations. So that it does not become a blocker for other repos.

File: i18n/config.py [hunk 2]
```diff
@@ -9,13 +9,13 @@
...
 # Typically this should be the 'edx-platform' directory.
-BASE_DIR = Path('.').abspath()  # pylint: disable=invalid-name
+BASE_DIR = Path('.').abspath()
 
 # The base filename for the configuration file.
 BASE_CONFIG_FILENAME = 'config.yaml'
 
 
-class Configuration(object):
+class Configuration:
```
Comment (jmbowman) [inline i18n/config.py]: This should keep the explicit `object` superclass until we've dropped Python 2.7 support.

File: i18n/converter.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 import itertools
 
 
-class Converter(object):
+class Converter:
```
Comment (jmbowman) [inline i18n/converter.py]: Same here.

**Comment by Ayub-Khan** [comment]
@jmbowman kindly review again.

**Pull request merged**